client/src/components/review.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import ReviewDisplay from './reviewDisplay.jsx'
+import ReviewRating from './reviewRating.jsx'
+
+class Review extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            hello: "Hello World"
+        }
+    }
+    render() {
+        console.log("Rendering Reviews")
+        console.log("Review.jsx props are: ", this.props.reviews)
+        return (
+            <div style={{width: 648}}>
+                <div className="separator-bar" style={ {marginTop: 24,marginBottom: 24} }>
+                    <hr style={{ color: '#EBEBEB', backgroundColor: '#EBEBEB', height: .0, borderColor : '#EBEBEB' }} />
+                </div>
+                <div style={{width: 648, height: 88}}>
+                    <h2 className="hello" style={ {color: '#484848', fontSize: 24, fontWeight: 600, fontFamily: "Circular, -apple-system, BlinkMacSystemFont, Roboto, Helvetica Neue, sans-serif", lineHeight: 1.25, paddingtop: 2, paddingBottom: 2,margin: 0, marginBottom: 20} }>Reviews</h2>
+                    <div style={ {height: 42, color: '#484848', fontSize: 18, fontWeight: 600, fontFamily: "Circular, -apple-system, BlinkMacSystemFont, Roboto, Helvetica Neue, sans-serif", lineHeight: 1.25, paddingtop: 2, paddingBottom: 2,margin: 0} }>
+                    *{this.props.fulldata.totalRatingAverages} | {this.props.fulldata.totalReviews} reviews <form className="searchbox">Search Placeholder</form>
+                    {/* *4.91      |       896 reviews <form className="searchbox">Search Placeholder</form> */}
+                    </div>
+                </div>
+                <div className="separator-bar" style={ {marginBottom: 15} }>
+                    <hr style={{ color: '#EBEBEB', backgroundColor: '#EBEBEB', height: .0, borderColor : '#EBEBEB' }} />
+                </div>
+                    <ReviewRating ratingAverages={this.props.ratingAverages}/>
+                <div className="spacing" style={ {marginTop: 10,marginBottom: 24} }>
+                </div>
+                <div>
+                    <ReviewDisplay reviews={this.props.reviews}/>
+                </div>
+            </div>
+        )
+    }
+}
+//648 px wide
+//this should be for display a single review
+// box-sizing: border-box;
+//     line-height: 1.43;
+//
+//margin-bottom: 24px;d
+
```

Comment:
You've left these two comment blocks for future use? if not it should be removed before merging into the master